### DISCUSSION TITLE
Select menu options clickable area

### DISCUSSION THREAD

**Discussion by bhamtil-twilio** [twilio-labs/paste#2727] [General] [answered]
Hello! My team has received customer feedback that the options in a select menu (accessed from a kebab menu button) are only clickable on the text itself while the customer was expecting to be able to click the whole row. This is a minor issue but a nuisance for them nonetheless. Is this how the select component is meant to behave, or can it be configured?

**Comment by gloriliale** [comment]
Hi @bhamtil-twilio!
That is unexpected; could you provide a link to a coded example of the option only having text clickable? The options in a menu are designed for users to be able to click the whole row, as you stated. 

I tried to reproduce it in this [code sandbox](https://codesandbox.io/s/menu-clickable-row-ib0ifq?file=/src/index.tsx), but it shows the whole row as being clickable (peek the console). Getting a snippet of the code would help figure out what's going awry. Many thanks!

**Reply by Aobasi4** [reply]
Hey, The issue maybe the connected anchor inside the menuitem but i'm not sure, here is the snippet
 <Box display="flex" alignItems="center" data-testid="create-alarm-menu">
      {errorCode ? (
        <>
          <MenuButton
            {...menu}
            variant="reset"
            size="reset"
            data-testid="create-alarm-menu-button"
          >
            <MoreIcon decorative={false} title="Options" />
          </MenuButton>
          <Menu {...menu} aria-label={labels.createAlarm}>
            <MenuItem {...menu}>
              <ConnectedAnchor
                href={routeFor("createAlarm") + `?errorCode=${errorCode}`}
                data-testid="create-alarm-for-error-code"
              >
                {labels.createAlarm}
              </ConnectedAnchor>
            </MenuItem>
            {renderInspectOption({})}
          </Menu>
        </>
      ) : null}
    </Box>

**Reply by nkrantz** [reply]
Hey @Aobasi4 - if you could, please share a CodeSandbox of the reproduction so that we can see the bug you're noticing. Feel free to use this [Paste starter template](https://codesandbox.io/s/paste-starter-template-rz6u3). Thanks!

**Reply by nkrantz** [reply]
Hi @bhamtil-twilio @Aobasi4 - are you still having trouble with Select? We don't support anchors inside of menuItems, so that's likely causing some issues, but please feel free to update this thread if you have more questions about it.

**Comment by shleewhite** [comment, marked as answer]
Marking this as answered due to inactivity. If you have follow up questions please let us know!
